soc/arm/nuvoton_npcx/common/reg/reg_def.h

```diff
@@ -1134,4 +1134,43 @@ struct smb_fifo_reg {
 #define NPCX_SMBRXF_CTL_RX_THR           FIELD(0, 6)
 #define NPCX_SMBRXF_CTL_LAST             7
 
+/*
+ * Internal 32-bit Timer (ITIM32) device registers
+ */
+struct itim32_reg {
+	volatile uint8_t reserved1;
+	/* 0x001: Internal Timer Prescaler */
+	volatile uint8_t ITPRE;
+	volatile uint8_t reserved2[2];
+	/* 0x004: Internal Timer Control and Status */
+	volatile uint8_t ITCTS;
+	volatile uint8_t reserved3[3];
+	/* 0x008: Internal 32-Bit Timer Counter */
+	volatile uint32_t ITCNT32;
```

Comment:
The NPCX7nxC datasheet shows all register names as ITPRE32, ITCTS32, and ITCNT32.  I'm not sure about other datasheets.  But I think it improves readability of the driver if the 32-bit timer always includes "32" on the register name and same with the 64-bit timer (add "64").

Reply:
The original thought is to keep the ITCNT register and ITCNT_XXX field as the same prefix no matter 32-bit or 64-bit timers. But from the improving readability point of view, I agree with you. I will change all register's names to meet our datasheet and modified the prefix of the field name from NPCX_ITCTS_ to NPCX_ITCTSXX_.

Reply:
Done.